### PR TITLE
Add typed properties and more typed parameters to methods

### PR DESCRIPTION
**Pull request by mmachatschek**
# Pull Request

## What does this PR do?
This adds PHP 7.4 typed properties and some types to methods.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: src/Delegates/HandlesSystem.php [hunk 1]
```diff
@@ -7,13 +7,11 @@
 use MeiliSearch\Endpoints\Health;
 use MeiliSearch\Endpoints\Keys;
 use MeiliSearch\Endpoints\Stats;
-use MeiliSearch\Endpoints\SysInfo;
```
Comment (mmachatschek) [inline src/Delegates/HandlesSystem.php:10]: I removed this, as there is no SysInfo class anymore

**Review by mmachatschek — COMMENTED** [review]
@alallema I will revert/undo the php7.4 version updates in the Github Action files and the composer file as soon, as PR #284 is merged

**Review by alallema — APPROVED** [review]
Thanks again @mmachatschek for this PR ✨ 
I fixed the conflict and added the types in the `TestCase` file hope it's ok for you.
LGTM!

**Comment by mmachatschek** [comment]
> I fixed the conflict and added the types in the TestCase file hope it's ok for you.

@alallema thanks for that, forgot about those types :+1:

**Comment by alallema** [comment]
bors merge

**Comment by alallema** [comment]
> @alallema thanks for that, forgot about those types 👍

No, it was not there, that's why. They arrived with the update of the branch I made 😉

**Pull request merged**